### PR TITLE
[All Hosts] (debugging) how to debug with Edge Chromium tools

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
Fixes [3040](https://github.com/OfficeDev/office-js-docs-pr/issues/3040)

Note to reviewers: Pressing F12 can open the tools in a dialog, but it doesn't work in a task pane. You have to use Ctrl-Shift-I.

### CODE REVIEW

File: docs/testing/debug-add-ins-using-devtools-edge-chromium.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+## Debug a task pane add-in using Microsoft Edge (Chromium-based) developer tools
+
+> [!NOTE]
+> If your add-in has an [add-in command](../design/add-in-commands.md) that executes a function, the function runs in a hidden browser process that the Microsoft Edge (Chromium-based) developer tools cannot be launched from, so the technique described in this article cannot be used to debug code in the function.
+
+1. [Sideload](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) and run the add-in.
+1. Run the Microsoft Edge (Chromium-based) developer tools by one of these methods:
+
+   - Be sure the add-in's task pane has focus and press **Ctrl-Shift-I**.
+   - Right-click the task pane to open the context menu, or open the [personality menu](../design/task-pane-add-ins.md#personality-menu), and select **Inspect**.
```
Comment (millerds) [inline docs/testing/debug-add-ins-using-devtools-edge-chromium.md:32]: I see inspect in the right click menu, but I see "Attach Debugger" in the personality menu.  The both (along with the shortcut) open up the developer tools as expected.

**Review by millerds — CHANGES_REQUESTED** [review]

File: docs/testing/debug-add-ins-using-devtools-edge-chromium.md (hunk 1 above)
Comment (Rick-Kirkham) [reply docs/testing/debug-add-ins-using-devtools-edge-chromium.md]: Good catch. Fixed.

**Review by millerds — APPROVED** [review]

File: docs/includes/identify-webview-in-ui.md [hunk 2]
```diff
@@ -1,3 +1 @@
...
-
-Note that right-clicking a task pane in an add-in opens a context menu if Microsoft Edge (Chromium) is being used. If it is not, then right-clicking does nothing.
+In recent versions of Office, one way to identify the webview control that Office is using is through the [personality menu](../design/task-pane-add-ins.md#personality-menu) on any add-in where it is available. (The personality menu is not supported in Outlook.) Open the menu and select **Security Info**. In the **Security Info** dialog on Windows, the **Runtime** reports **Microsoft Edge**, **Microsoft Edge Legacy**, or **Internet Explorer**. The runtime isn't included on the dialog in older versions of Office.
```
Comment (ElizabethSamuel-MSFT) [inline docs/includes/identify-webview-in-ui.md:1]:
```suggestion
In recent versions of Office, one way to identify the webview control that Office is using is through the [personality menu](../design/task-pane-add-ins.md#personality-menu) on any add-in where it's available. (The personality menu is not supported in Outlook.) Open the menu and select **Security Info**. In the **Security Info** dialog on Windows, the **Runtime** reports **Microsoft Edge**, **Microsoft Edge Legacy**, or **Internet Explorer**. The runtime isn't included on the dialog in older versions of Office. 
```

File: docs/testing/debug-add-ins-using-devtools-edge-chromium.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## Debug a task pane add-in using Microsoft Edge (Chromium-based) developer tools
+
+> [!NOTE]
+> If your add-in has an [add-in command](../design/add-in-commands.md) that executes a function, the function runs in a hidden browser process that the Microsoft Edge (Chromium-based) developer tools cannot be launched from, so the technique described in this article cannot be used to debug code in the function.
+
+1. [Sideload](create-a-network-shared-folder-catalog-for-task-pane-and-content-add-ins.md) and run the add-in.
+1. Run the Microsoft Edge (Chromium-based) developer tools by one of these methods:
+
+   - Be sure the add-in's task pane has focus and press **Ctrl-Shift-I**.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-chromium.md:31]:
```suggestion
   - Be sure the add-in's task pane has focus and press **Ctrl+Shift+I**.
```

File: docs/testing/debug-add-ins-using-devtools-edge-chromium.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+1. Open the file that you want to debug with the following steps.
+
+   1. On the far right of the tool's top menu bar, select the **...** button and then select **Search**.
+   1. Enter a line of code from the file you want to debug in the search box. It should be something that's not likely to be in any other file.
+   1. Select the refresh button.
+   1. In the search results, select the line to open the code file in the pane above the search results.
+
+   :::image type="content" source="../images/open-file-in-edge-chromium-devtools.png" alt-text="Screenshot of Edge Chromium developer tools source tab with 4 parts labelled A through D.":::
+
+1. To set a breakpoint, select the the line number of the line in the code file. A red dot appears by the line in the code file. In the debugger window to the right, the breakpoint is registered in the **Breakpoints** drop down.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-chromium.md:44]:
```suggestion
1. To set a breakpoint, select the line number of the line in the code file. A red dot appears by the line in the code file. In the debugger window to the right, the breakpoint is registered in the **Breakpoints** drop down.
```

File: docs/testing/debug-add-ins-using-devtools-edge-chromium.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+## Debug a dialog in an add-in
+
+If your add-in uses the Office Dialog API, the dialog runs in a separate process from the task pane (if any) and the tool must be started from that separate process. Follow these steps.
+
+1. Run the add-in.
+1. Open the dialog and be sure it has focus.
+1. Open the Microsoft Edge (Chromium-based) developer tools by one of these methods:
+
+   - Press **Ctrl-Shift-I** or **F12**.
```
Comment (ElizabethSamuel-MSFT) [inline docs/testing/debug-add-ins-using-devtools-edge-chromium.md:58]:
```suggestion
   - Press **Ctrl+Shift+I** or **F12**.
```

**Review by ElizabethSamuel-MSFT — APPROVED** [review]
Looks good. Just a couple of tweaks.

**Pull request merged**